From c17bb8a334d3fded3f9e88fa37ada55f9cb80efb Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Fri, 7 Sep 2018 09:52:49 +0200
Subject: [PATCH] Correct the Replaces in emacs-{gtk,lucid,nox}

The predecessors of emacs-gtk, emacs-lucid and emacs-nox in
src:emacs25 were called emacs25, emacs25-lucid and emacs25-nox
respectively, so replace those rather than the new variants.

Closes: 907997
---
 debian/control | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/debian/control b/debian/control
index 6d4a533ea20..1dcd6a2a00a 100644
--- a/debian/control
+++ b/debian/control
@@ -42,7 +42,7 @@ Provides: emacs, emacsen, editor, info-browser, mail-reader, news-reader
 Suggests: emacs-common-non-dfsg
 Conflicts: emacs-gtk, emacs-nox
 Replaces: emacs-gtk, emacs-nox,
-  emacs-gtk (<< 1:25), emacs-lucid (<< 1:25), emacs-nox (<< 1:25)
+  emacs25 (<< 1:25), emacs25-lucid (<< 1:25), emacs25-nox (<< 1:25)
 Description: GNU Emacs editor (with Lucid GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs with support for a graphical user
@@ -74,7 +74,7 @@ Provides: emacs, emacsen, editor, info-browser, mail-reader, news-reader
 Suggests: emacs-common-non-dfsg
 Conflicts: emacs-gtk, emacs-lucid
 Replaces: emacs-gtk, emacs-lucid,
-  emacs-gtk (<< 1:25), emacs-lucid (<< 1:25), emacs-nox (<< 1:25)
+  emacs25 (<< 1:25), emacs25-lucid (<< 1:25), emacs25-nox (<< 1:25)
 Description: GNU Emacs editor (without GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs compiled without support for X,
@@ -100,7 +100,7 @@ Provides: emacs, emacsen, editor, info-browser, mail-reader, news-reader
 Suggests: emacs-common-non-dfsg
 Conflicts: emacs-lucid, emacs-nox
 Replaces: emacs-lucid, emacs-nox,
-  emacs-gtk (<< 1:25), emacs-lucid (<< 1:25), emacs-nox (<< 1:25)
+  emacs25 (<< 1:25), emacs25-lucid (<< 1:25), emacs25-nox (<< 1:25)
 Description: GNU Emacs editor (with GTK+ GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs with a graphical user interface
-- 
2.19.0.rc2

